### PR TITLE
[Server][DBTableMonitoring] Add a update processing of hostname.

### PR DESCRIPTION
**Pull request by Mnakagawa**

### CODE REVIEW

File: server/src/DBTablesMonitoring.cc [hunk 1]
```diff
@@ -2101,8 +2099,16 @@ void DBTablesMonitoring::updateHosts(const HostInfoList &hostInfoList,
 	HostInfoListConstIterator newHostsItr = hostInfoList.begin();
 	for (; newHostsItr != hostInfoList.end(); ++newHostsItr) {
 		const HostInfo &newHostInfo = *newHostsItr;
-		if (currValidHosts.erase(newHostInfo.id) >= 1) {
-			// The host already exits. We have nrothing to do.
+		HostIdHostInfoMapIterator currHostItr = currValidHosts.find(newHostInfo.id);
+		if (currHostItr != currValidHosts.end()) {
+			const HostInfo &existHost = *currHostItr->second;
+			if (existHost.hostName != newHostInfo.hostName){
+				updatedHostInfoList.push_back(newHostInfo);
```
Comment (kz0817) [inline server/src/DBTablesMonitoring.cc]: I believe we can call updatedHostInfoList.push_back() once.
As a result, the code will be simple.

**Comment by Mnakagawa** [comment]
kz0817 san
Thank you your pointed out.
I fixed a algolythm.

File: server/src/DBTablesMonitoring.cc [hunk 2]
```diff
@@ -2101,9 +2099,15 @@ void DBTablesMonitoring::updateHosts(const HostInfoList &hostInfoList,
 	HostInfoListConstIterator newHostsItr = hostInfoList.begin();
 	for (; newHostsItr != hostInfoList.end(); ++newHostsItr) {
 		const HostInfo &newHostInfo = *newHostsItr;
-		if (currValidHosts.erase(newHostInfo.id) >= 1) {
-			// The host already exits. We have nrothing to do.
-			continue;
+		HostIdHostInfoMapIterator currHostItr = currValidHosts.find(newHostInfo.id);
+		if (currHostItr != currValidHosts.end()) {
+			const HostInfo &existHost = *currHostItr->second;
```
Comment (kz0817) [inline server/src/DBTablesMonitoring.cc]: 'exist' is a verb. The combination 'exist' and 'host' is unnatural.
How about using 'existingHost'.
Or 'currHost' which is an opposite word of newHost(Info).

**Comment by Mnakagawa** [comment]
@kz0817 san
I rename existHost to currHost.

**Pull request merged**